sympy/integrals/integrals.py

```diff
@@ -532,6 +533,38 @@ def try_meijerg(function, xab):
                             function = ret
                             continue
 
+            if not isinstance(antideriv,Integral) and antideriv is not None:
+                sym = xab[0]
+                if antideriv.is_Add:
+                    terms = antideriv.args
+                else:
+                    terms = tuple((antideriv,))
+                for term in terms:
+                    atan_part = term.atoms(atan)
+                    for atan_term in atan_part:
+                        atan_arg = atan_term.args[0]
+                        # Checking `atan_arg` to be linear combination of `tan` or `cot`
+                        if atan_arg.has(tan):
+                            x1 = Dummy('x1')
+                            tan_part = atan_arg.atoms(tan).pop()
```

Comment:
Could there be more than one `tan`?

Reply:
Should I check `len(atan_arg.atoms(tan)) == 1` ?

Reply:
I think it should be investigated if several `tan` terms (in a single `atan_arg`) could also be handled in the same way (each one with its own `floor(a - pi/2)`). One could then take
`for tan_part in atan_arg.atoms(tan):`
instead of `if atan_arg.has(tan):`.

Reply:
yup.